.github/workflows/wheels.yml

```diff
@@ -112,7 +112,7 @@ jobs:
         path: dist
 
     - name: Generate artifact attestation for sdist and wheels
-      uses: actions/attest-build-provenance@v1
+      uses: actions/attest-build-provenance@v2
```

Comment:
Looks valid in the basic use-case still: https://github.com/actions/attest-build-provenance